Login

rafimuhammadi opened the pull request:
Login Functionality Design, Form handling,  Validation, and post data to the backend are done.

rafimuhammadi committed:
Login

Login Functionality Design, Form handling,  Validation, and post data to the backend are done.

Linda-Vu commented:
@rafimuhammadi let's get these conflicts resolved and we can review your PR again! ping your teammates in the channel to see if they can pair program with you on it.

rafimuhammadi committed:
Login, Registration.

Changing the design of the login page and also applying changes to the registration page for form handling, data validation and also form submitting.

rafimuhammadi committed:
Merge branch 'dev' of https://github.com/Code-the-Dream-School/bb-practicum-team-4-front into Abdul-Rafi-Login-component

rafimuhammadi committed:
Update Logo.jsx

rafimuhammadi committed:
new changes done

rafimuhammadi commented:
> @rafimuhammadi let's get these conflicts resolved and we can review your PR again! ping your teammates in the channel to see if they can pair program with you on it.
Done

rafimuhammadi commented:
I solved the conflicts and also applied the new changes on the login and registration form.

rafimuhammadi committed:
login page

Creating a Public Layout for the login and register page added a login link on the header component and a background image for the login page.

rafimuhammadi committed:
remove unnecessary file from components

rafimuhammadi committed:
Merge branch 'dev' into Abdul-Rafi-Login-component

rafimuhammadi committed:
integrating of login and register components with the backend

rafimuhammadi committed:
Merge branch 'Abdul-Rafi-Login-component' of https://github.com/Code-the-Dream-School/bb-practicum-team-4-front into Abdul-Rafi-Login-component

rafimuhammadi committed:
Nave link and install react external link
